basecoin/app/Cargo.toml

```
@@ -0,0 +1,41 @@
+[package]
+name       = "basecoin-app"
+version     = { workspace = true }
+edition     = { workspace = true }
+license     = { workspace = true }
+readme      = { workspace = true }
+repository  = { workspace = true }
+authors     = { workspace = true }
+description  = """
+    Contains a rudimentary implementation of the Tendermint ABCI application.
```

```suggestion
    Contains a simple implementation of the Tendermint ABCI application.
```

Just a suggestion. You can ignore this.

---

[ed23af3](https://github.com/informalsystems/basecoin-rs/pull/156/commits/ed23af315bc0755b0de03f466ae89a9f5b6d6a31)